templates/projects.html

```diff
@@ -60,11 +61,30 @@
       margin: 0;
       font-weight: bold;
     }
+
+  #context-bar {
+    background-color: white !important;
+  }
 </style>
 {% endblock %}
 
 <!-- Remove contextbar from top of page -->
-{% block contextbar %}{% endblock %}
+{% block contextbar %}
```

Comment:
Per above, better to keep this in body and not put it in contextbar but then not use contextbar's styles. I'm committing a change to put this back in body.